src/llmq/quorums.cpp

```diff
@@ -197,8 +209,10 @@ void CQuorumManager::TriggerQuorumDataRecoveryThreads(const CBlockIndex* pIndex)
 
         // First check if we are member of any quorum of this type
         bool fWeAreQuorumTypeMember{false};
-        for (const auto& pQuorum : vecQuorums) {
-            if (pQuorum->IsValidMember(activeMasternodeInfo.proTxHash)) {
+
+        auto proTxHash = WITH_LOCK(activeMasternodeInfoCs, return activeMasternodeInfo.proTxHash);
+        for (const auto &pQuorum : vecQuorums) {
```

Comment:
```suggestion
        for (const auto& pQuorum : vecQuorums) {
```

>Align pointers and references to the left i.e. use `type& var` and not `type &var`.

https://github.com/dashpay/dash/blob/master/doc/developer-notes.md#coding-style-c

Reply:
applied